msteams-platform/toolkit/TeamsFx-manifest-customization.md

```diff
@@ -42,14 +38,7 @@ During local debug, Teams Toolkit loads manifest from `manifest.local.template.j
         }
         ```
 
-    Besides each config placeholder in `manifest.remote.template.json`, there is a `Go to config file`. You can navigate to configuration file by selecting it.
-
-## Supported placeholder in manifest.local.template.json
-
-`{{localSettings.xx}}` is pre-defined placeholder whose value is resolved by Teams Toolkit, defined in `localSettings.json`. Ensure not to modify the values in localSettings.json.
-
- > [!NOTE]
- > Ensure not to customize the local manifest.
+    Besides each config placeholder in `manifest.template.json`, there is a `Go to config file` or `View the state file`. You can navigate to configuration file by selecting it.
```

Comment:
With LiuNing and Yefu's new feature, this UI changes and should we change as well this time?

Reply:
Their change should be in preview features, can update when it's public.